index.html

```diff
@@ -5,12 +5,43 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Hello microverse</title>
-    <link rel="stylesheet" href="index.css" />
-
+    <link rel="stylesheet"  href="assets/css/smallscreen.css">
+    <link rel="stylesheet"  href="assets/css/widescreen.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
 </head>
 
 <body>
-    <h1 class='hello-microvers-title'>Hello microverse</h1>
+    <!-- Header -->
+    <div class="header">
+        <a href="#" class="logo">My Logo</a>
+        <a href="#" class="icon" >
+            <i class="fa-solid fa-bars"></i>
+        </a>
+    </div>
+    <!-- end of header -->
+    <!-- landing page -->
```

Comment:
- [ ] [OPTIONAL] Please, try to remove comments, to make your code more clear 👍

Reply:
Thanks for you comment and approval